gulpfile.babel.ts

```diff
@@ -341,84 +318,106 @@ gulp.task('types', async () => {
   await exec('yarn typedefs');
 });
 
+let config: WebpackTaskOptions;
+const remoteCompatConfig = {
+  afterBuild: setupExamples,
+} as const;
+
 if (args.remote) {
-  gulp.task('sdk', () => {
-    return webpackTask({
-      devtool: 'remote',
-      disableMinification: true,
-      entry: './src/inboxsdk-js/inboxsdk-REMOTE',
-      destName: sdkFilename,
-      standalone: 'InboxSDK',
-    });
-  });
-  gulp.task(
-    'remote',
-    gulp.series('pageWorld', function impBundle() {
-      return webpackTask({
-        devtool: 'remote',
-        entry: './src/platform-implementation-js/main-INTEGRATED-PAGEWORLD',
-        destName: 'platform-implementation.js',
-        afterBuild: setupExamples,
-      });
-    })
-  );
-  gulp.task('default', gulp.series('sdk', 'remote'));
+  config = {
```

Comment:
We should inline the block from 405 invoking the webpack task both here and for the integratedPageWorld branch.

Reply:
@Macil mentioned that https://github.com/InboxSDK/InboxSDK/pull/948#discussion_r1289181171 would assuage his concerns here.